### PR TITLE
Update-FMLA-Content

### PR DESCRIPTION
**Pull request by jamiealbrecht**
setting the right expectations around FMLA

### CODE REVIEW

File: pages/leave-policies.md [hunk 1]
```diff
@@ -21,5 +21,7 @@ Both parents may use their annual or sick leave after the birth of a child, for
...
 
-**In addition** to the paid leave described above, employees can take leave without pay (under FMLA) for up to 12 weeks per calendar year. You can take this leave within one year of the birth of your child. You do not need to take it all at once. Employees are eligible to take intermittent leave within the first year of their child’s birth. For instance, you could take unpaid leave on Tuesdays and Thursdays for several weeks in a row. 
+**In addition** to the paid leave described above, employees can take leave without pay (under the Family and Medical Leave Act) for up to 12 weeks per calendar year. An employee is entitled to taking FMLA with supervisor approval and when they have been an employee of the federal government for at least 12 months. You can take this leave within one year of the birth of your child. You do not need to take it all at once. Employees are eligible to take intermittent leave within the first year of their child’s birth. For instance, you could take unpaid leave on Tuesdays and Thursdays for several weeks in a row. 
+
```
Comment (fureigh) [inline pages/leave-policies.md]: @jamiealbrecht Nbd but I'm gonna take out these extra lines.

File: pages/leave-policies.md [hunk 2]
```diff
@@ -21,5 +21,7 @@ Both parents may use their annual or sick leave after the birth of a child, for
...
 
-**In addition** to the paid leave described above, employees can take leave without pay (under FMLA) for up to 12 weeks per calendar year. You can take this leave within one year of the birth of your child. You do not need to take it all at once. Employees are eligible to take intermittent leave within the first year of their child’s birth. For instance, you could take unpaid leave on Tuesdays and Thursdays for several weeks in a row. 
+**In addition** to the paid leave described above, employees can take leave without pay (under the Family and Medical Leave Act) for up to 12 weeks per calendar year. An employee is entitled to taking FMLA with supervisor approval and when they have been an employee of the federal government for at least 12 months. You can take this leave within one year of the birth of your child. You do not need to take it all at once. Employees are eligible to take intermittent leave within the first year of their child’s birth. For instance, you could take unpaid leave on Tuesdays and Thursdays for several weeks in a row.
```
Comment (fureigh) [inline pages/leave-policies.md]: @jamiealbrecht: I'm slightly confused by the "An employee is entitled to taking FMLA with supervisor approval" sentence. (Also trying to navigate the 'employees'-vs.'you' discrepancy... let's standardize on one later.) Attempting a rephrase:

"Once you've been an employee of the federal government for at least 12 months, you're entitled to take FMLA, provided your supervisor approves it."

Two questions:
1. Is that rephrase accurate?
2. Which is more accurate, 'entitled' or 'eligible'?

**Comment by jamiealbrecht** [comment]
i think eligible is better.

**Comment by fureigh** [comment]
@jamiealbrecht Cool. I updated the text — take another look and merge if it looks good to you!

**Comment by fureigh** [comment]
high-five-aladdin

**Pull request merged**